content/ux-strat-bootcamp/part-one/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Part One

```

Comment:
This file shouldn't be listed on the syllabus page.  Only the individual tasks. Otherwise people will just get confused. Cards will move very slowly and managers will not have resolution on what people are working on unless they ask.

Reply:
I shall remove it then. I'm assuming this applies to 'part-two' as well? My question is: How then do we make a separation between the two parts/ phases of bootcamp? Will it then just be 10 tasks to achieve during bootcamp?